src/app/local-storage-manager.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { Todo } from './interfaces/todo';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Rather than providing this as part of the composition root, we should inject it via the providers in the module (which you have) so you can remove this

Reply:
removed the root reference